### PR TITLE
feat: migrate to dedicated AWS account and update CI

### PR DESCRIPTION
**Pull request by xav**
# Description

Migrate to the dedicated AWS account and switch CI to the [shared flows](https://github.com/WalletConnect/ci_workflows)

Resolves # (issue)

## How Has This Been Tested?

Deployed manually

## Due Diligence

* [ ] Breaking change
* [ ] Requires a documentation update
* [ ] Requires a e2e/integration test update

### CODE REVIEW

**Review by arein — APPROVED** [review]
LGTM!
Just out of extra caution: please add on statuspage and let's confirm metrics/alarms afterwards

File: .github/workflows/debug_validate.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+name: ⚙️ Debug - Validate
+on:
+  push:
+    branches:
+      - 58-migrate-ti-dedicated-aws-account
```
Comment (xDarksome) [inline .github/workflows/debug_validate.yml:5]: We probably need to change this to smth?

**Review by xDarksome — APPROVED** [review]
Looks good, assuming:
- CI/CD actions are tested
- integration tests pass
- analytics export works

File: .github/workflows/debug_validate.yml (hunk 1 above)
Comment (xav) [reply .github/workflows/debug_validate.yml:5]: this was just a hack to force the tests to run, we can remove it

**Pull request merged**